### PR TITLE
AIRNEIS-55: Product Page on Mobile

### PR DESCRIPTION
**Pull request by hallycsse**
image
Waiting for endpoint api

### CODE REVIEW

**Comment by cedricgautier** [comment]
Are you able to fetch the backend ?

File: mobile/src/screens/ProductScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  name: "Canapé Blanc",
+  thumbnailURL: "https://picsum.photos/400/300",
+  outOfStock: false,
+  materials: ["Wood", "Coton"],
+  produitsSimilaires: [{ name: "Canapé Marron", id: 2, thumbnailURL: "https://picsum.photos/400/300", priceWithTaxes: 1450 }],
+  description:
+    "Lorem ipsum dolor sit amet. Aut nihil Quis et dolorum quis hicsint voluptas id aspernatur corporis.ad voluptatem aliquam. At autem molestias et sequi harum aut rerum quia est  consequatur cupiditate et aliquid rerum et labore  dignissimos?",
+  images: ["https://picsum.photos/400/300", "https://picsum.photos/400/400"],
+  priceWithTaxes: 1200
+}
```
Comment (cedricgautier) [inline mobile/src/screens/ProductScreen.jsx]: Are you able to fetch the backend ?

Comment (barnabepilliaudin) [reply mobile/src/screens/ProductScreen.jsx]: @cedricgautier done

Comment (barnabepilliaudin) [reply mobile/src/screens/ProductScreen.jsx]: There is still some sample data for things that the api doesn't manage yet

**Review by cedricgautier — APPROVED** [review]

**Pull request merged**